styles/colors-gallium-day.less

```diff
@@ -126,7 +126,8 @@
 
 // LabeledItem
 // ---------------------------------------
-@agate-labelediconbutton-color: @agate-accent;
+@agate-labelediconbutton-label-color: @agate-white;
```

Comment:
Should be `@agate-text-color`? The text is currently unreadable in sampler for all skins other than gallium-night and other components (e.g. Item) use text color as the unfocused color.

Reply:
We have to be extra careful when making additions/changes to gallium day until the base skin PR (#147) lands. Until then, we need to manually copy the old values from gallium-day into each of the other color files to prevent visually altering them.

Reply:
Sounds like @teckliew should review that first so we can get it landed and then fix up this. Yes?

Reply:
Yes. That will help with several new/updated Agate components. However, if it's projected to take a long time to review, longer than it would take to add these values, these could get done and updated later.